dm/dm-precheck.md

```diff
@@ -90,6 +90,30 @@ For the full data migration mode (`task-mode: full`), in addition to the [common
 
     - If sharded tables have auto-increment primary keys, the precheck returns a warning. If there are conflicts in auto-increment primary keys, see [Handle conflicts of auto-increment primary key](/dm/shard-merge-best-practices.md#handle-conflicts-of-auto-increment-primary-key) for solutions.
 
+#### Check items for physical import
+
+If you set `import-mode: "physical"` in the task configuration, the following check items are added to ensure that [Physical Import](/tidb-lightning/tidb-lightning-physical-import-mode.md) runs normally. After following the prompts, if you find it difficult to meet the requirements of these check items, you can try to use the [logical import mode](/tidb-lightning/tidb-lightning-logical-import-mode.md) to import data.
+
+* Empty Regions in the downstream database
+
+    - If the number of empty Regions is greater than `max(1000, 3 * the number of tables)` (the larger of "1000" and "3 times the number of tables"), the precheck returns a warning. You can adjust related PD parameters to speed up the merging of empty Regions and wait for the number of empty Regions to decrease. See [PD Scheduling Best Practices - Slow Region Merge](/best-practices/pd-scheduling-best-practices.md#region-merge-is-slow).
+
+* Region distribution in the downstream database
+
+    - Checks the number of Regions in different TiKV nodes. If the node with the least number of Regions has a number of Regions that is 75% of the node with the most number of Regions, the precheck returns a warning. You can adjust related PD parameters to speed up the scheduling of Regions and wait for the number of Regions to change. See [PD Scheduling Best Practices - Leader/Region distribution is not balanced](/best-practices/pd-scheduling-best-practices.md#leadersregions-are-not-evenly-distributed).
+
+* The version of TiDB, PD, and TiKV in the downstream database
+
+    - Physical import must call the interfaces of TiDB, PD, and TiKV. If the version is not compatible, the precheck returns an error.
+
+* The free space of the downstream database
+
+    - Estimates the sum of the sizes of all tables in the allow list in the upstream database (`source_size`). If the free space of the downstream database is less than `source_size`, the precheck returns an error. If the free space of the downstream database is less than the number of TiKV replicas \* `source_size` \* 2, the precheck returns a warning.
+
+* Whether the downstream database is running tasks that are incompatible with physical import
+
+    - Currently, physical import is incompatible with [TiCDC](/ticdc/ticdc-overview.md) and [PITR](/br/br-pitr-guide.md) tasks. If these tasks are running in the downstream database, the precheck returns an error.
```

Comment:
```suggestion
    - Currently, physical import is incompatible with [TiCDC](/ticdc/ticdc-overview.md) and [PiTR](/br/br-pitr-guide.md) tasks. If these tasks are running in the downstream database, the precheck returns an error.
```

maybe it's "Point in TIme Recovery"

Reply:
Actually, this has been discussed a lot. 😂 
We used "PiTR" in the first place but later switched to "PITR", as it is consistent with [Wikipedia](https://en.wikipedia.org/wiki/Point-in-time_recovery) and [AWS documentation](https://aws.amazon.com/cn/dynamodb/pitr/).